src/DataStructure/Collection.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Platformsh\Client\DataStructure;
```

Comment:
Hmmm I intended `DataStructure` for very abstract things (it was for `ReadOnlyStructureTrait`), and this class is api-specific.

Reply:
I'm pretty bad at naming things. Ideas?